cmd/ttn-lw-stack/commands/config.go

```diff
@@ -60,6 +64,8 @@ var DefaultConfig = Config{
 	JS:          shared_joinserver.DefaultJoinServerConfig,
 	Console:     shared_console.DefaultConsoleConfig,
 	GCS:         shared_gatewayconfigurationserver.DefaultGatewayConfigurationServerConfig,
+	DTC:         shared_devicetemplateconverter.DefaultDeviceTemplateConverterConfig,
```

Comment:
Let's maintain the same order here as in `Config` above.
Is `QRG` missing as well?